### ISSUE TITLE
Missing annotations in Form when searching annotations

### ISSUE THREAD

**Issue by pgaudet** [geneontology/noctua-form#192]
Hi, 

With the Swiss-Prot curators we noticed that when using the Search Annotation menu, not all annotations are returned - for example for human YAP (UniProt:P46937)

<img width="987" alt="image" src="https://user-images.githubusercontent.com/4782928/175269983-6fe18d4b-5d22-407c-b307-aa674e0bcd19.png">

However there are annotations missing 
for example 
[transcription coactivator activity](http://amigo.geneontology.org/amigo/term/GO:0003713) IDA from [PMID:18280240](http://amigo.geneontology.org/amigo/reference/PMID:18280240) is missing. This is an annotation that was done is 2010; shouldn't it be there? 

Thanks, Pascale

@lionelbreuza
@pmasson55
@vanaukenk 
@tmushayahama

**Comment by kltm** [comment]
@tmushayahama They both update either monthly of every other week. If there something specific you're digging after?

**Comment by pgaudet** [comment]
This impacts usability - all annotations should be available

**Comment by vanaukenk** [comment]
I've done some testing this morning to see if I can understand what the problem might be.  

I tested searching existing MF annotations for a random set of genes/gene products for different species, with or without isoforms, and I can't find any missing annotations for any of them except human YAP1 (UniProtKB:P46937).

I also specifically tested other genes/gene products with annotations to 'transcription coactivator activity' and _can_ find those when searching their MF annotations.

Lastly, since it's still available, I tried the Function Companion workbench in the graph editor and there I can find the missing human YAP1 MF annotations and import them successfully.  The Assigned By field in the Noctua form table isn't displaying correctly for annotations imported with the Function Companion, but I don't know if that's at all related to the reported issue here.  I suspect not.

Bottom line - I'll need to do some more systematic testing with the input of GO software developers to understand what's going on here.

**Comment by kltm** [comment]
@vanaukenk Should we chat about this at some point? What would be the best way forward here?

**Comment by kltm** [comment]
Taking a quick look at this with @pgaudet, it looks like the form is contacting the GOlr instance and finding docs, but they not seem to be rendered? I'm wondering if this is a "bug" bug (and maybe is an "ongoing" issue) or something that needs to be more systematically looked at?

**Comment by pgaudet** [comment]
@tmushayahama Can you please have a look ?

**Comment by tmushayahama** [comment]
Aghhh it's a very simple hiccup, apologies couldn't find it earlier. Thanks for the above comments/discussion. After some debugging, having exactly 100 results was a little suspicious. Then I remember, when we initially designed the search database for Noctua Form 1, we put limit of 100. so I believed it escaped all these years.

![image](https://user-images.githubusercontent.com/2273929/207782868-db67fd32-cc3c-4d09-80a7-d5c20f04ee77.png)

And for sure the count is 149 same as amigo
![image](https://user-images.githubusercontent.com/2273929/207782921-465d656b-3c93-4a87-b2ef-e12f531fbfa6.png)

So will just remove the limit. Actually, maybe increase the limit to some large number or lets think of a way. Pagination is not a viable options because results are not folded
tagging @vanaukenk @kltm @pgaudet @ukemi

**Comment by pgaudet** [comment]
Hi @tmushayahama , thanks for finding this!! What's weird is that we see fewer than 100 results (but maybe that's the same problem). 

I think we need to remove the limits, otherwise it defeats the purpose of that functionality 
@vanaukenk @ukemi

**Comment by pgaudet** [comment]
Can we prioritize this ? this is really impeding work.

**Comment by vanaukenk** [comment]
2023-03-23 workbenches call: we will try to get this fix in for tonight's maintenance outage.

**Comment by vanaukenk** [comment]
Testing after the 2023-03-23 Noctua maintenance outage:

I tested all MF annotations in AmiGO for human YAP1 (UniProtKB:P46937) and all of the terms, with the correct number of annotations, are now there in the search.

I also tested a subset of mouse Ctnnb1 BP annotations and everything looks okay there, too.
